.github/run_browserstack_acceptance.sh

```diff
@@ -5,5 +5,6 @@ GITHUB_BRANCH=${GITHUB_REF#refs/heads/}
 export BROWSERSTACK_BUILD_ID="${GITHUB_BRANCH} - ${GITHUB_RUN_ID}"
 COMMIT_MSG_TITLE=$(git log -n 1 --pretty=format:%s)
 export BROWSERSTACK_TEST_RUN_NAME=$COMMIT_MSG_TITLE
+export IS_DEVELOPMENT_PREVIEW="false"
```

Comment:
Did we ever figure out why the tests were running in dev mode? By default, it should be Prod

Reply:
I mostly added this here for clarity. Why did we believe they were running in dev mode again? When I look at the acceptance tests for the PR where I removed postcss, they successfully ran in IE11 in browserstack which isn't possible if it were running in dev mode